api/src/main/java/org/apache/cloudstack/api/response/NetworkResponse.java

```diff
@@ -190,6 +190,10 @@
     @Param(description = "VPC the network belongs to")
     private String vpcId;
 
+    @SerializedName(ApiConstants.TUNGSTEN_VIRTUAL_ROUTER_UUID)
+    @Param(description = "Tungsten virtual router the network belongs to")
```

Comment:
Again, end user docs should have full name of "tungsten fabric"

Reply:
```suggestion
    @Param(description = "Tungsten-Fabric virtual router the network belongs to")
```